packaging/miden-faucet.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Miden faucet
+Wants=network-online.target
+
+[Install]
+WantedBy=multi-user.target
+
+[Service]
+Type=exec
+Environment="RUST_LOG=info"
+ExecStart=miden-faucet start --config /etc/miden/miden-faucet.toml
+WorkingDirectory=/opt/miden
```

Comment:
I believe the faucet and node need separate working directories as they both write to disk. So probably we need

```
/opt/miden/miden-node/
/opt/miden/miden-faucet/
```

Reply:
Updated the service files to reflect this